### PR TITLE
Correcting identifiers

### PR DESCRIPTION
**Pull request by lovesprung**
Correcting identifiers to follow the guide "Do not surround IP addresses, domain names or any other identifyers with backticks. Use italics (* *) to mark any inline IP address, domain name, file name, file location or similar." https://docs.nephio.org/docs/guides/contributor-guides/documentation/

### CODE REVIEW

File: content/en/docs/porch/package-variant.md [hunk 1]
```diff
@@ -1114,103 +1114,103 @@ spec:
...
-| upstream       | The upstream PackageRevision.                                |
+| upstream       | The upstream *PackageRevision*.                                |
 | repository     | The downstream Repository.                                   |
 | target         | The target object (details vary; see below).                 |
 
-There is one expression that is an exception to the table above. Since the `repository` value corresponds to the
-Repository of the downstream, we must first evaluate the `downstream.repoExpr` expression to *find* that repository. 
-Thus, for that expression only, `repository` is not a valid variable.
+There is one expression that is an exception to the table above. Since the repository value corresponds to the
+Repository of the downstream, we must first evaluate the `ownstream.repoExpr expression to find that repository.
```
Comment (liamfallon) [inline content/en/docs/porch/package-variant.md:1133]:
```suggestion
Repository of the downstream, we must first evaluate the `downstream.repoExpr expression to find that repository. 
```

**Comment by liamfallon** [comment]
Nice work! Just 2 tiny typos.

**Comment by lovesprung** [comment]
@liamfallon thanks! corrected the suggestion + one more small thing

**Pull request merged**